drivers/wifi/nrf700x/src/fmac_main.c

```diff
@@ -713,6 +713,11 @@ static struct wifi_mgmt_ops nrf_wifi_mgmt_ops = {
 	.reg_domain = nrf_wifi_reg_domain,
 	.get_power_save_config = nrf_wifi_get_power_save_config,
 #endif /* CONFIG_NRF700X_STA_MODE */
+#ifdef CONFIG_NRF700X_RAW_DATA_TX
+	.mode = nrf_wifi_mode,
+	.filter = nrf_wifi_filter,
```

Comment:
This should not be protected with `#ifdef CONFIG_NRF700X_RAW_DATA_TX`

Reply:
okay

Reply:
filter setting is removed from raw data tx changes